build.gradle

```diff
@@ -69,7 +69,7 @@ sourceCompatibility = 1.7
 
 compileJava {
   options.warnings = false
-  options.compilerArgs.addAll(['--release', '8'])
+  //options.compilerArgs.addAll(['--release', '8'])
```

Comment:
I am going to put this back.  I took it out because I was not able to build at all with it in.  I am getting this error with it in "invalid flag: --release".  I am wondering if adding this is necessary if we are doing targetCompatibility = 1.7 and sourceCompatibility = 1.7.